lib/p2p/Pool.ts

```diff
@@ -423,7 +427,8 @@ class Pool extends EventEmitter {
       this.peers.add(peer);
 
       // request peer's orders and known nodes
-      peer.sendPacket(new packets.GetOrdersPacket());
+      const getOrdersPacketBody: packets.GetOrdersPacketBody = { pairIds: this.handshakeData.pairs };
+      peer.sendPacket(new packets.GetOrdersPacket(getOrdersPacketBody));
```

Comment:
No need for 2 separate lines here. you can create the packet body as the argument